src/app/features/products/views/ProductsHeader.tsx

```diff
@@ -0,0 +1,26 @@
+import { ConnectBanner } from '~/src/app/shared/animations/connectBanner';
+import * as tv from '../ProductsTV';
+
+const renderRosarivoText = (text: string) => (
+  <span className={tv.ProductsRosarivoTextTv()}>{text}</span>
+);

```

Comment:
Apesar de entender do que se trata, o nome da função e da classe é pouco sugestivo, talvez algo como:
```suggestion
const renderHighlightedText= (text: string) => (
  <span className={tv.ProductsHighlightTextTv()}>{text}</span>
);

```

Acha que faz sentido?
Se sim, teria que alterar também as classes.